package.json

```diff
@@ -19,6 +19,7 @@
         "bootstrap": "^4.4.1",
         "chart.js": "^2.9.3",
         "exif-js": "^2.3.0",
+        "file-type": "^14.3.0",
```

Comment:
add license declaration in ThirdPartyNotice.txt

Reply:
actually we are not using this librarary anymore - I'll delete it

Reply:
done